services/QuillLMS/app/models/feedback_history.rb

```diff
@@ -331,7 +331,8 @@ def self.get_total_count(activity_id: nil, start_date: nil, end_date: nil, filte
     query = FeedbackHistory.apply_activity_session_filter(query, filter_type) if filter_type
     query = query.having(FeedbackHistory.responses_for_scoring) if responses_for_scoring
     query = query.where("feedback_histories.activity_version = ?", activity_version) if activity_version
-    query.length
+    result = ActiveRecord::Base.connection.execute("SELECT COUNT(*) from (#{query.to_sql}) as nickname")
```

Comment:
Do we need `as nickname`?

Reply:
We need  `as <literally any string>`, to avoid a BigQuery syntax error